azure-pipelines-ci.yml

```diff
@@ -0,0 +1,103 @@
+# This is the main CI pipeline.
+# It lints, builds and runs unit tests for all of our projects.
+
+# Trigger this pipeline on all pushes landing on our primary integration branches.
+trigger:
+  branches:
+    include:
+    - main
+    - develop
+    - release/*
+
+# Trigger this pipeline on PRs against our primary integration branches.
+pr:
+  - main
+  - develop
+  - release/*
+
+pool:
+  vmImage: 'macos-latest'
+
+steps:
+- task: servicetree-link-build-task@1
+  displayName: 'ServiceTree Integration'
+  inputs:
+    ServiceTreeGateway: 'ServiceTree Gateway'
+    Service: 'e312ec77-9c77-48a7-9837-32fe6d08d460'
+    BuildOutputUsage: 'test'
```

Comment:
is it safe to share this information as public? is it possible to use secret inside azure devops to get these values?

Reply:
Yup, something we're already doing in other Azure repositories.